src/App.jsx

```diff
@@ -1,40 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
according to task requirements, you should save only the `id`:
![image](https://github.com/mate-academy/react_tabs-js/assets/86923089/3f83649c-93c4-43a0-aa4a-ad1ad6b07509)

in this case, you need a function that finds tab by id in all tabs array
then pass only `id` (that you saved here) to `Tabs` component, and use there the same function to get the selected tab and show content

Reply:
I think the solution you propose has worse performance than storing the entire tabs at once. To implement the task requirements, I suggest just introducing a variable that stores the selectedTabId:
`const selectedTabId = selectedTab.id;`